.github/workflows/test-e2e.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     needs: e2e
     runs-on: ubuntu-latest
     container:
-      image: cypress/browsers:node14.15.0-chrome86-ff82
+      image: vertigostudio/cypress-ff72
```

Comment:
if we have this image do we still need all the other docker files from this pr ? @gutoslv

Reply:
No, I've just removed them on the last amend from the commit. Thank you for noticing that.